### ISSUE TITLE
Input prefix wraps if too long

### ISSUE THREAD

**Issue by Larsklopstra** [laravel-filament/filament#307]
**Describe the bug**
If you have a rather _large_ prefix it will wrap.

**To reproduce**

```php
Components\TextInput::make('slug')->prefix('This prefix shouldn\'t wrap but it still does... :('),
```

**Expected behavior**
It should not wrap.

**Screenshots**
![image](https://user-images.githubusercontent.com/25669876/113000982-a2492780-9170-11eb-8b05-a46f6d9e673f.png)

**Context**
N/A

**Additional details**
Should be fixed with a simple `whitespace-nowrap`

**Comment by yob-yob** [comment]
I don't know how this would be a good UI if the input is too small because of the Prefix..

in the `Expected behavior` section, you could put in a screenshot that shows a TextInput with a prefix that is long enough to cause a wrap then based on your `additional details` section, you can edit the input prefix to have a `whitespace-nowrap` style.

in this way, we can see and decide if your proposed style is a good User Interface to be implemented.

**Comment by danharrin** [comment]
@ryanscherler Do you have any thoughts about the proposed change and the concerns raised by @yob-yob?

**Comment by Larsklopstra** [comment]
I think the middle approach would to not break "single words" `my-cool-domain.com/`

**Comment by ryanscherler** [comment]
I like the middle approach with not breaking single words as noted by @Larsklopstra

**Comment by danharrin** [comment]
Closed by #372.
